test/system/group_offers_test.rb

```diff
@@ -185,4 +185,14 @@ def setup
     assert select_values.include? volunteer_one.id
     assert select_values.include? volunteer_two.id
   end
+
+  test 'department_manager cannot access group offer pages unless there is a department assigned' do
+    department_manager = create :user, role: 'department_manager'
```

Comment:
`create :department_manager`

Reply:
not here, because `create :department_manager` creates a user that has allready a department associated